gemrb/plugins/SDLVideo/SDL20Video.cpp

```diff
@@ -731,8 +731,29 @@ int SDL20VideoDriver::ProcessEvent(const SDL_Event & event)
 				break;
 			}
 			else {
-				// we do not want these events to cascade down to SDL_KEYDOWN, so we return here instead of at default .
-				return SDLVideoDriver::ProcessEvent(event);
+				/**
+				 * As being SDL2-only, try to query the clipboard state to
+				 * paste when middle clicking the mouse.
+				 */
+				if (
+					   event.button.button == SDL_BUTTON_MIDDLE
+					&& event.type == SDL_MOUSEBUTTONDOWN
+					&& SDL_HasClipboardText()
+				) {
+					const char *pasteValue = SDL_GetClipboardText();
+
+					if (pasteValue != NULL) {
+						String *pasteValueString = StringFromCString(pasteValue);
```

Comment:
Oh wait, this does not make me happy with umlaute and stuff alike.

Reply:
Well, we should live with this for now: Typing non-ASCII values into the console already produces garbage anyway, so ...